static/app/components/events/interfaces/spans/inlineDocs.tsx

```diff
@@ -134,4 +236,8 @@ const DocumentationWrapper = styled('div')`
   }
 `;
 
-export default withApi(InlineDocs);
+const StyledCodeSnippet = styled(CodeSnippet)`
+  margin-bottom: ${space(3)};
+`;
```

Comment:
nit: should not this be after the export?

Reply:
is there a guideline for it? Searching the codebase it seems like it is done differently in each file 🤔